src/modules/navigator/mission.cpp

```diff
@@ -237,6 +238,10 @@ Mission::update_offboard_mission()
 
 	if (orb_copy(ORB_ID(offboard_mission), _navigator->get_offboard_mission_sub(), &_offboard_mission) == OK) {
 		warnx("offboard mission updated: dataman_id=%d, count=%d, current_seq=%d", _offboard_mission.dataman_id, _offboard_mission.count, _offboard_mission.current_seq);
+
+		/* inform the user on succesful storing of mission */
+		mavlink_log_info(_navigator->get_mavlink_fd(), "Mission succesfully stored: %d items", _offboard_mission.count);
```

Comment:
There is no need for this. The GCS complains if storing the mission failed. Don't start to spam the user.